ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualSignedTransactionVerifier.kt

```diff
@@ -0,0 +1,13 @@
+package net.corda.v5.ledger.consensual.transaction
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.Suspendable
+
+@CordaSerializable
+fun interface ConsensualSignedTransactionVerifier {
+
+    // Should this return a Try.Success/Error? Or control purely through exceptions, but this means that should not
+    // be used to determine behaviour in the peer flow.
```

Comment:
Good question - what is the desired behaviour to tell the initiating party "We're not going to sign this"?

Reply:
Can we have a discussion about this.